cob_bringup/robots/cob4-5.xml

```diff
@@ -261,7 +263,12 @@
 		</group>
 
 		<include file="$(find cob_bringup)/drivers/mimic.launch">
-			<arg unless="$(arg sim)" name="rotation" value="270"/>
+			<arg unless="$(arg sim)" name="rotation" value="0"/>
```

Comment:
`<arg unless="$(arg sim)" name="rotation" value="270"/>` is used for the other robots
does this apply to `cob4-5` as well?